packages/form-state-manager/src/files/form-manager-context.ts

```diff
@@ -13,6 +13,7 @@ const FormManagerContext = createContext<ManagerContextValue>({
   handleSubmit: noop,
   registerField: noop,
   unRegisterField: noop,
```

Comment:
It does not belong to this PR... but unregister should be one word `unregisterField`. :smile:

Reply:
yup I think this whole state will move to the global manager API. And the context will be eventually updated only when subscribed.